cmd/vic-machine/create/create.go

```diff
@@ -491,6 +499,7 @@ func (c *Create) Run(cliContext *cli.Context) error {
 	ctx, cancel := context.WithTimeout(context.Background(), c.Timeout)
 	defer cancel()
 
+	c.Data.ScratchSize = c.scratchSize
```

Comment:
@emlin is there a better place for me to do this assignment? This moves the input from the user from VirtualContainerHostConfigSpec to InstallerData and while this location _works_ I don't know that it's the best place, as it's a seemingly-random assignment in this location (to someone unfamiliar with the code at least)